### DISCUSSION TITLE
How do we run with hybrid vs isopycnic BLOM i NorESM2.5?

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#3] [BLOM grid and resolution] [answered]
Hi,
Can someone please explain how we activate the hybrid version of BLOM (I'm assuming isopycnic still is the default option, but I might be wrong - then please correct me :) in NorESM2.5? @matsbn @oyvindseland @mvertens

**Comment by matsbn** [comment, marked as answer]
BLOM isopycnic is still default. To activate hybrid vertical coordinate one needs to execute
```
./xmlchange BLOM_VCOORD=cntiso_hybrid,BLOM_TURBULENT_CLOSURE=
```
before case.setup (I suggest to make this selectable in the compset definition, so that e.g. BLOM%HYB will set these options https://github.com/NorESMhub/BLOM/issues/361).

With BLOM hybrid and the tnx1v4 grid, I recommend the following consistent initial conditions for CICE and BLOM:

In user_nl_cice, set
```
ice_ic = "/cluster/work/users/matsbn/archive/NOIIAJRAOC20TR_TL319_tn14_ppm_notz_20240115/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_ppm_notz_20240115.cice.r.1775-01-01-00000.nc”
```
In user_nl_blom, set
```
ICFILE = "/cluster/work/users/matsbn/archive/NOIIAJRAOC20TR_TL319_tn14_ppm_notz_20240115/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_ppm_notz_20240115.blom.r.1775-01-01-00000.nc”
```

<br>
With BLOM isopycnic, I recommend the following initial conditions for CICE and BLOM, that has a similar spinup procedure as above for BLOM hybrid:

In user_nl_cice, set
```
ice_ic="/cluster/work/users/matsbn/archive/NOIIAJRAOC20TR_TL319_tn14_20190710/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_20190710.cice.r.1775-01-01-00000.nc"
```
In user_nl_blom, set
```
ICFILE   = "/cluster/work/users/matsbn/archive/NOIIAJRAOC20TR_TL319_tn14_20190710/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_20190710.micom.r.1775-01-01-00000.nc"
```
